### ISSUE TITLE
Elevated circulating pyridinoline concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6026]
**Preferred term label:** Elevated circulating pyridinoline concentration

**Synonyms** Elevated circulating PYD concentration

**Definition (free text, please give PubMed ID)** Concentration of pyridinoline in the blood is above the normal range., PMID: 23729802; PMID: 9284202

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Pyridinoline (PYD) and deoxypyridinoline (DPD) are formed as the cross-links between collagen molecules during collagen maturation; thus their presence in circulation is the result of collagen degradation. Their presence in the blood circulation is a marker of bone turnover.

**Comment by pnrobinson** [comment]
http://purl.obolibrary.org/obo/CHEBI_88539

**Comment by pnrobinson** [comment]
added
